inst/rmarkdown/templates/cholera_outbreak/skeleton/skeleton.Rmd

```diff
@@ -444,22 +501,31 @@ unique(linelist_cleaned$sex)
 ##
 
 
+# Drop unused rows  ---------------------------------
+# Your data might have rows that are blank besides ID number.
+# The below keeps variables only with a case ID and an admission date.
+linelist_cleaned <- linelist_cleaned %>% 
+  filter(!is.na(case_number) & !is.na(date_of_admission)) 
+
+
 # Date variables ---------------------------------
 # IF YOU ARE USING THE DATA DICTIONARY USE THIS OTHERWISE UNCOMMENT LINES BELOW
 # make sure all date variables are formatted as dates 
 DATEVARS <- filter(linelist_dict, data_element_valuetype == "DATE") %>% 
   select(data_element_shortname) # select date vars
```

Comment:
we might want to add a filtering step and replace `select()` with `pull()`

```r
DATEVARS <- filter(linelist_dict, data_element_valuetype == "DATE") %>%
  filter(data_element_shortname %in% names(linelist_clean)) %>% # filter to match the column names of your data
  pull(data_element_shortname) 
```

Note, however, this will change all downstream instances of `DATEVARS$data_element_shortname` to `DATEVARS`

Reply:
@kdoyle514, Please let me know if you would like me to change these and I'll make the commits.